views/lib/narrow-view.js

```diff
@@ -7,28 +7,20 @@ import NarrowViewGroup from './narrow-view-group'
 export default class NarrowView {
   /**
    * Initializes a narrow view.
-   * @param {number} groupQty - A number of visible groups (sub tables) within a narrow view.
-   * @param {Column[]} columns - Table columns.
-   * @param {Row[]} rows - Table rows.
-   * @param {Row[]} headers - Table headers.
-   * @param {number} titleColumnQty - Number of title columns in a table.
+   * @param {Table} table - An inflection table object.
    */
-  constructor (groupQty, columns, rows, headers, titleColumnQty) {
-    this.columns = columns
-    this.rows = rows
-    this.headers = headers
-    this.titleColumnQty = titleColumnQty
+  constructor (table) {
+    this.table = table
     this.groups = []
-    this.groupQty = groupQty
     this.groupSize = 0
-    if (groupQty) {
-      this.groupSize = this.columns.length / groupQty
+    if (this.table.features.firstColumnFeature.size) {
```

Comment:
can you add a comment that explains what this is doing?

Reply:
Good idea! Updated.